ui/redux/actions/livestream.js

```diff
@@ -175,91 +104,115 @@ const findActiveStreams = async (
   // Find the first upcoming claim (if one exists) for each channel that's actively broadcasting a stream.
   const upcomingClaims = await dispatch(fetchUpcomingLivestreamClaims(channelIDs, lang));
 
-  // Filter out any of those claims that aren't scheduled to start within the configured "soon" buffer time (ex. next 5 min).
+  // Filter out any of those claims that aren't scheduled to start within the configured "soon" buffer time (ex. next 15 min).
   const startingSoonClaims = filterUpcomingLiveStreamClaims(upcomingClaims);
 
   // Reduce the claim list to one "live" claim per channel, based on how close each claim's
   // release time is to the time the channels stream started.
-  const allClaims = Object.assign({}, mostRecentClaims, startingSoonClaims);
+  const allClaims = Object.assign(
+    {},
+    mostRecentClaims,
+    !isEmpty(startingSoonClaims) ? startingSoonClaims : upcomingClaims
+  );
 
   return determineLiveClaim(allClaims, liveChannels);
 };
 
-export const doFetchChannelLiveStatus = (channelId: string) => {
-  return async (dispatch: Dispatch) => {
-    try {
-      const { channelStatus, channelData } = await fetchLiveChannel(channelId);
+export const doFetchChannelLiveStatus = (channelId: string) => async (dispatch: Dispatch) => {
+  const statusForId = `channel-live-status`;
```

Comment:
@saltrafael I went with a single key - only time this breaks is if you have multiple livestreams in various tabs. If we put in a claim id, we'll continue filling up local storage with claim ids if a user navigates away. Not sure how to clean those up.

The other strange edge case was in pop out mode, it stops the interval I believe and doesn't end nicely. Think I'm okay with this, but would be also just nice to dispose the player? If you go back into the claim, then it takes 2 loops. 

I also noticed the stream end scenario calls the live api like 3 times - probably not a big deal, but maybe there's something obvious?